test/smoke-fails/targetid_multi_image/Makefile

```diff
@@ -0,0 +1,21 @@
+include ../../Makefile.defs
+
+TESTNAME     = targetid-multi-image
+TESTSRC_MAIN = targetid-multi-image.c
+TESTSRC_AUX  =
+TESTSRC_ALL  = $(TESTSRC_MAIN) $(TESTSRC_AUX)
+
+OMP_FLAGS    = -fopenmp -fopenmp-targets=amdgcn-amd-amdhsa,amdgcn-amd-amdhsa -Xopenmp-target=amdgcn-amd-amdhsa -march=gfx90a:xnack- -Xopenmp-target=amdgcn-amd-amdhsa -march=gfx90a:xnack+
```

Comment:
I may be wrong but I believe only a single triple in -fopenmp-targets=amdgcn-amd-amdhsa is required?

Reply:
Done